Xamarin.Forms.Platform.WPF/FormsContentLoader.cs

```diff
@@ -54,11 +54,17 @@ private object CreateOrResizeContent(FrameworkElement parent, VisualElement visu
 			//if (Debugger.IsAttached)
 			//	Console.WriteLine("Page type : " + visualElement.GetType() + " (" + (visualElement as Page).Title + ") -- Parent type : " + visualElement.Parent.GetType() + " -- " + parent.ActualHeight + "H*" + parent.ActualWidth + "W");
 
-			visualElement.Layout(new Rectangle(0, 0, parent.ActualWidth, parent.ActualHeight));
+			var actualRect = new Rectangle(0, 0, parent.ActualWidth, parent.ActualHeight);
+			visualElement.Layout(actualRect);
+
+			// ControlTemplate adds an additional layer through which to send sizing changes.
+			var contentPage = visualElement as ContentPage;
```

Comment:
shouldn't you cast to `IControlTemplated` ? or are you 100% sure that this only needs to be done on ContentPages ?

Reply:
`IControlTemplated` does not define the `Content` property.
Only on `ContentPage` has `Content` property which we need.

Reply:
what about ContentView ?